### PR TITLE
allow dual branding only after binlookup

### PR DESCRIPTION
**Pull request by erenbesel**

### CODE REVIEW

**Review by descorp — APPROVED** [review]

**Review by nauaros — APPROVED** [review]
Nice!

File: AdyenCard/Utilities/BinLookup/BinLookupResponse.swift [hunk 1]
```diff
@@ -20,13 +20,18 @@ internal struct BinLookupResponse: Response {
...
 
+    /// :nodoc:
+    internal var isCreatedLocally: Bool = false
+    
     /// :nodoc:
     internal init(brands: [CardBrand]? = nil,
                   requestId: String = UUID().uuidString,
-                  issuingCountryCode: String? = "NL") {
+                  issuingCountryCode: String? = "NL",
+                  isCreatedLocally: Bool = true) {
```
Comment (mohammedDehairy) [inline AdyenCard/Utilities/BinLookup/BinLookupResponse.swift:30]: it should be set to false when coming from bin lookup, right? I don't see where its being set to false?

Comment (erenbesel) [reply AdyenCard/Utilities/BinLookup/BinLookupResponse.swift:30]: When from binlookup, this init won't be called. And the default value is `false`. Should be ok, no?

Comment (mohammedDehairy) [reply AdyenCard/Utilities/BinLookup/BinLookupResponse.swift:30]: ah, I see, its fine, but its a bit unclear that that's the case from first glance :-)

**Pull request merged**